src/components/project/list-filter.query.ts

```diff
@@ -6,17 +6,19 @@ export const projectListFilter = (input: ProjectListInput) =>
   filter.builder(input.filter, {
     type: filter.skip, // already applied
     status: filter.stringListProp(),
-    onlyMultipleEngagements: ({ value, query }) => {
-      if (!value) return null;
-      query
-        .match([
-          node('node'),
-          relation('out', '', 'engagement', ACTIVE),
-          node('engagement', 'Engagement'),
-        ])
-        .with('*, count(engagement) as engagementCount');
-      return { engagementCount: greaterThan(1) };
-    },
+    onlyMultipleEngagements:
+      ({ value, query }) =>
+      () =>
```

Comment:
This applies the where clause after the others, as I'm sure you've figured out. But the `with node, engagementCount` will still drop the `requestingUser` variable. Have you tried running this query?

Reply:
Yes, I tested it out and works as expected even using the combination of other filters like `pinned` and `mine`.

Reply:
Yes I just checked as well and `requestingUser` isn't re-used outside of filters. It's re-matched again, which is fine.